assets/css/styles.css

```diff
@@ -96,6 +96,20 @@ a:hover,
   margin: 30px 0;
 }
 
+.announcement {
+  display: inline-block;
```

Comment:
I tested this and you don't need display, height, margin, padding, vertical-align. Can you remove and re-test after you move the announcement to the right place in the DOM?

Reply:
yep ur right, removed